netweaver/install_ascs.sls

```diff
@@ -22,7 +24,7 @@ netweaver_install_{{ instance_name }}:
   netweaver.installed:
     - name: {{ node.sid.lower() }}
     - inst: {{ instance }}
-    - password: {{ node.master_password }}
+    - password: {{ netweaver.master_password }}
```

Comment:
Is this the `master_password` or the `sidadm` password? I would say the 2nd, but I'm not sure

Reply:
The same for the other sap instances